### ISSUE TITLE
App shortcut icon not working

### ISSUE THREAD

**Issue by otiskujawa** [Geoxor/Amethyst#435]
**Describe the bug**
Icon is not showing on desktop shortcut created by installer
![image](https://user-images.githubusercontent.com/49094075/219132339-8a71d904-d9dd-426a-b123-45a81267da68.png)

**Amethyst Details (required):**
 - Version 1.8.55 on windows 11

**To Reproduce (optional)**
1. Install amethyst using .exe installer
2. Look at shortcut on desktop

Icon on taskbar is ok
![image](https://user-images.githubusercontent.com/49094075/219132931-0e638974-0bf0-45b8-bda0-071033f9f89d.png)

Similar behavior can be observed on linux task bar but this need to be checked and added as another issue

**Comment by LayreBoi** [comment]
Try to check if the file path of the shortcut exists. The icon on the taskbar is showing the amethyst logo because the icon is from the program itself. The shortcut in the desktop is created by the setup. You can change the file path to the existing amethyst program manually, but for now idk.

**Comment by Geoxor** [comment]
![image](https://user-images.githubusercontent.com/34042825/228916088-44396492-6cd9-43f6-adae-4e3bef348b5c.png)
i have no clue what this error means

**Comment by LayreBoi** [comment]
The setup seems to have put these characters as the path for the icon, which Windows obviously can't find as the path doesn't exist. The (temporary) fix is to change the icon path to the Amethyst's exe or ico by clicking on "Change Icon" and locate Amethyst (C:\Users\{username}\AppData\Local\Programs\amethyst\Amethyst.exe). Temporary, because the setup might change the icon path after update until the issue is resolved. I have no clue what's the cause of this.

**Comment by Geoxor** [comment]
Yeah I think this is some [@electron-builder](https://github.com/electron-userland/electron-builder) bug because it appeared out of nowhere after a dependency update.

**Comment by LettuceGameStudios** [comment]
From what i can see it appears to be a description length issue

**Comment by LettuceGameStudios** [comment]
latest version?

**Comment by LettuceGameStudios** [comment]
Fixed in [https://github.com/Geoxor/Amethyst/commit/678ed07118f60e106d0408208ebbe9436ffbf4a4](url)

**Comment by LettuceGameStudios** [comment]
@Geoxor
